packages/jest-config/src/index.ts

```diff
@@ -289,7 +293,10 @@ export function readConfigs(
     }
   }
 
-  if (projects.length > 1) {
+  if (
+    projects.length > 1 ||
+    (projects.length && typeof projects[0] === 'object')
```

Comment:
I think there is actually a deeper issue here that will require more than this quick fix, since the types seem convinced it's a `string` when my understanding (from it's use at FB) is that it can be a string or a config.

However, this solves the immediate issue.

Reply:
yup, from #5176